src/Microsoft.TemplateEngine.Authoring.TemplateVerifier/TemplateVerifierOptions.cs

```diff
@@ -88,6 +94,26 @@ public class TemplateVerifierOptions : IOptions<TemplateVerifierOptions>
         /// </summary>
         public VerifyDirectory? CustomDirectoryVerifier { get; private set; }
 
+        /// <summary>
+        /// Gets a custom prefix prepended in front of generated scenario name - result used for naming verification subdirectories.
```

Comment:
Not sure if I got the meaning of this doc.
```suggestion
        /// Gets the custom scenario name; if specified it will be used as part of verification snapshot.